protected/helpers/EndNoteHelper.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * To generate citation in customized EndNote format in xml schema
+ *
+ * Class EndNoteHelper
+ *
+ * The followings are the available model relations:
+ * @property Dataset $dataset
+ *
+ */
+
+class EndNoteHelper
+{
+    public static function getRecords($full_doi)
+    {
+        $splitFull = explode("/", $full_doi);
+        $identifier = $splitFull[1];
+        $dataset = Dataset::model()->findAllByAttributes(array("identifier"=>'100002'));
```

Comment:
Hi @pli888 

I want to access the `Dataset` model and then extract attributes for building the `EndNote` citation. But the `$dataset` output is empty. I have tried different `find` methods, the `$dataset` is also empty.  Can you have a look for me?

Reply:
The `$dataset` output is empty because there is no dataset with `identifier=100002`. Data for unit tests come from files in the fixtures directory. Try the following in `EndNoteHelper.php` class:
```
$dataset = Dataset::model()->findAllByAttributes(array("identifier"=>'100243'));
```

Reply:
I tried to use the `identifier id` from the `dataset` in `fixture directory`, the `$dataset` output is still empty.

Reply:
Maybe you need to declare a fixtures array so the dataset table has some data for you to use. Add this to your `EndNoteHelperTest` class:
```
protected $fixtures=array(
    'dataset'=>'Dataset',
);
```